receiver/hostmetricsreceiver/internal/scraper/virtualmemoryscraper/pagefile_windows.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
+
+package virtualmemoryscraper
+
+import (
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	modPsapi           = windows.NewLazySystemDLL("psapi.dll")
+	procEnumPageFilesW = modPsapi.NewProc("EnumPageFilesW")
+)
```

Comment:
Ideally everything in this file will get upstreamed to `gopsutil`. I'll create a PR in that repo as well and update here if that gets merged.